docs/user_guide/mitigation.rst

```diff
@@ -194,6 +207,684 @@ Postprocessing
 
 .. currentmodule:: fairlearn.postprocessing
 
+What is a decision threshold?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Many postprocessing algorithms make decisions based on thresholds. But what
+is a decision threshold?
+
+In binary classification, the decision threshold is a number which represents the border between
+positive and negative predictions. In many implementations of classification algorithms, the
+default decision threshold is 0.5, which means that a sample with a probability to get 1
+(obtained by calling :code:`estimator.predict_proba()[:,1]`) of 0.6 will be classified
+as 1, and a sample with probability to get 1 of 0.4 will be classified as 0.
+If this threshold would be changed to 0.7, both samples would be classified
+as 0.
+
+Thresholder
+~~~~~~~~~~~~
+
+What it does
+^^^^^^^^^^^^
+:class:`fairlearn.postprocessing.Thresholder` is a post processing technique
+which outputs a binary prediction based on group-specific thresholds. The
+specified thresholds can work in two directions
+(:ref:`positive<pos_thresholds>`/:ref:`negative<negative_thresholds>`), and can
+be applied to the output of the :code:`predict()` function of a regressor,
+the second element of each prediction obtained from :code:`predict_proba()` of a classifier, or on
+:code:`decision_function()` of, for example, :class:`sklearn.svm.SVC` or
+:class:`sklearn.linear_model.LogisticRegression`. Examples of these different applications
+are presented below.
+
+Why use it
+^^^^^^^^^^^^
+:class:`Thresholder` is an easy to interpret tool, that can help explore
+trade-offs between fairness metrics and predictive performance. Where an algorithm like
+:class:`ThresholdOptimizer` “blindly” satisfies
```

Comment:
Instead of "blindly" I would suggest using "simply" or "merely."
From a cursory search I'd say the internet isn't quite in unanimous agreement about this yet, but there are people who have called the use of "blindly" as an adverb ableist. That was my hunch, too. I learned quite a bit about this topic recently and was shocked how many expressions that plenty of people (including myself) use quite frequently are ableist. So much to learn, so little time 🙁 

This may be a short starter on the topic: https://languageplease.org/ableism-ableist-language/